deploy/crds/kabanero_v1alpha1_kabanero_crd.yaml

```diff
@@ -122,6 +122,19 @@ spec:
               type: object
             version:
               type: string
+            webhook:
```

Comment:
These same updates need to be made to registry/manifests/kabanero-operator/0.3.0/kabanero_kabanero_crd.yaml

Reply:
Thanks for catching this.  My intention in the Makefile was to copy the current version of the CRD to the `registery/manifests/kabaner-operator/currentver/` directory, and then when that version is "final", copy the CRD over and move the current version to the next version.  The build does copy the file, but it looks like I also checked in that file at some point.  I will update it, but in the future this should not be necessary.